### ISSUE TITLE
Style/TernaryParentheses and ranges

### ISSUE THREAD

**Issue by dreyks** [bbatsov/rubocop#3511]
## False positive 
```ruby
(0..5).exclude?(3) ? 'yay' : 'nay'
```
```console
$ rubocop
```
### Expected behavior
Nothing is reported
### Actual behavior
`Omit parentheses for ternary conditions.`

The [regex](https://github.com/bbatsov/rubocop/blob/master/lib/rubocop/cop/style/ternary_parentheses.rb#L94) should probably be changed. I will look into this

## Faulty auro-correct
```ruby
(0..5).exclude?(3) ? 'yay' : 'nay'
```
```console
$ rubocop --auto-correct
undefined method `type' for :exclude?:Symbol
/bundle/gems/rubocop-0.43.0/lib/rubocop/cop/style/ternary_parentheses.rb:117:in `method_call_argument'
```

No idea on how to fix this, too much AST-magic for me

## RuboCop version
```console
$ rubocop -V
0.43.0 (using Parser 2.3.1.3, running on ruby 2.3.1 x86_64-linux)
```

**Comment by bbatsov** [comment]
That's why one shouldn't use regular expressions to detect expression boundaries. :-)
